reference/catalog.v3.yml

```diff
@@ -19594,6 +19596,8 @@ definitions:
   productVariant_Post_Product:
     description: |
       The model for a POST to create variants on a product.
+    type: array
```

Comment:
The way product_Post is structured, I don't think we should be changing productVariant_Post_Product to an array.